internal/node/node.bzl

```diff
@@ -155,6 +155,10 @@ def _nodejs_binary_impl(ctx):
         if k in ctx.var.keys():
             env_vars += "export %s=\"%s\"\n" % (k, ctx.var[k])
 
+    if ctx.attr.coverage:
+        # TODO: use proper path for tmpdir
+        env_vars += "export NODE_V8_COVERAGE=/tmp/$RANDOM\n"
```

Comment:
coverage only applies to test right? bazel gives you `$TEST_TMPDIR` but you should wait until you're inside the test action to try to read the environment

Reply:
While only doing coverage under `bazel test` makes sense, we often run out test targets with `bazel run` when we're developing them, simply becuase it's the same command we use to run our nodejs targets.
However, if there's no way to get access to a tmpdir under `bazel run` then we'll have to do that, we'll have to ensure this is documented

Reply:
Also note even with the `bazel coverage` integration, we'd still need this set since the v8 coverage format is only an intermediate format

Reply:
since we're only doing coverage with `bazel coverage` as per the below comment, I changed this to be `        env_vars += "export NODE_V8_COVERAGE=$TEST_TMPDIR/$RANDOM\n"` - not sure if $RANDOM is correct but it's close if not